From 3b5a15a3b09787f5beacd3fe8bb06e80f926eeaf Mon Sep 17 00:00:00 2001
From: Tomasz Figa <t.figa@samsung.com>
Date: Fri, 1 Mar 2013 15:27:59 +0100
Subject: [PATCH 0153/1302] ARM: dts: exynos4210-trats: Add physical display
 size properties

This patch adds physical size properties to panel node.

Signed-off-by: Tomasz Figa <t.figa@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4210-trats.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/exynos4210-trats.dts b/arch/arm/boot/dts/exynos4210-trats.dts
index a7ed9af..6b7dd39 100644
--- a/arch/arm/boot/dts/exynos4210-trats.dts
+++ b/arch/arm/boot/dts/exynos4210-trats.dts
@@ -511,6 +511,8 @@
 		video-source = <&dsi_0>;
 		flip-horizontal;
 		flip-vertical;
+		samsung,panel-width-mm = <58>; /* FIXME */
+		samsung,panel-height-mm = <103>; /* FIXME */
 
 		display-timings {
 			native-mode = <&timing0>;
-- 
1.8.3.2

